src/views/profile/bookmarks-view/helpers/render-bookmark-cards.tsx

```diff
@@ -0,0 +1,49 @@
+import { getTutorialSlug } from 'views/collection-view/helpers'
+import { formatTutorialData } from 'lib/learn-client/api/tutorial/formatting'
+import { ProductUsed } from 'lib/learn-client/types'
+import { ApiBookmark } from 'lib/learn-client/api/api-types'
+import TutorialCard from 'components/tutorial-card'
+import getReadableTime from 'components/tutorial-meta/components/badges/helpers'
+import { BookmarkButtonIconOnly } from 'components/bookmark-button'
+
+/**
+ * TODO: the bookmark button `handleClick` should
+ * render a confirmation dialog that can update data
+ */
+
+export default function renderBookmarkCard(bookmark: ApiBookmark) {
+	const {
+		id,
+		slug,
+		description,
+		readTime,
+		handsOnLab,
+		video,
+		name,
+		collectionCtx,
+		productsUsed,
+	} = formatTutorialData(bookmark.tutorial)
+	const duration = getReadableTime(readTime)
+	return (
+		<li key={id}>
+			<TutorialCard
+				heading={name}
+				description={description}
+				url={getTutorialSlug(slug, collectionCtx.default.slug)}
```

Comment:
What if a user has bookmarked the tutorial in the context of the non-default collection? Does this always link to the tutorial in the default collection?

Reply:
It does, this is how bookmarks currently behave in learn. There isn't collection associated data with the bookmark model in the API right now so we have no way of knowing the collection context of the tutorial at the time of bookmarking. We could note this as a future improvement.

Reply:
Got it. Thank you!